packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -114,7 +114,7 @@ export class PrepareNextSlotScheduler {
         // the slot 0 of next epoch will likely use this Previous Root Checkpoint state for state transition so we transfer cache here
         // the resulting state with cache will be cached in Checkpoint State Cache which is used for the upcoming block processing
         // for other slots dontTransferCached=true because we don't run state transition on this state
-        {dontTransferCache: !isEpochTransition},
+        {dontTransferCache: !isEpochTransition, asyncShufflingCalculation: true},
```

Comment:
add more comments for this flag referring to the issue

Reply:
updated